dev/com.ibm.ws.concurrent.mp.1.0/resources/com/ibm/ws/concurrent/mp/resources/CWWKCMessages.nlsprops

```diff
@@ -22,7 +22,7 @@
 #   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
 #   (single quote must be coded as one single quote '). 
 
-# All messages must use the range CWWCK1150 to CWWCK1199 except those specifically identified as moved
```

Comment:
was this an intentional change?

Reply:
Yes, to reserve this portion of the messages for the com.ibm.ws.concurrent.mp.cdi bundle